src/utils/blockchainExplorer.js

```diff
@@ -84,7 +84,7 @@ export async function viewTransactionOnBlockchain(chain: Chain, transaction: Vie
       routeName: WALLETCONNECT_BROWSER,
       params: {
         url: explorerLink,
-        title: t('title.chain_explorer', { title: upperFirst(chain) }),
+        title: t('title.chain_explorer', { title: ((chain === CHAIN.XDAI)) ? 'Gnosis' : upperFirst(chain) }),
```

Comment:
Please added title for both testnet and mainnet.
Like: isProdEnv() ? t('chains.xdai') : t('chains.sokol')
For reference: src/utils/uiConfig.js